datasets/br/br_slavery/crawler.py

```diff
@@ -0,0 +1,48 @@
+import csv
+from typing import Dict
+
+from rigour.mime.types import CSV
+
+from zavod import Context, helpers as h
+
+
+def crawl_row(context: Context, row: Dict[str, str]):
+    entity = context.make("Company")
+    entity.id = context.make_slug(row.pop("ID"))
+    entity.add("country", "br")
+
+    entity.add("name", row.pop("Empregador"))
+    entity.add("address", row.pop("Estabelecimento"))
+    entity.add("taxNumber", row.pop("CNPJ/CPF"))
+    # The Ministry of Labor and Social Security is taking "fiscal action"
+    entity.add("topics", "reg.action")
```

Comment:
I just want to confirm before we merge - if inclusion in this list automatically means being banned from public procurement, we should also add the `debarment` topic and explain in the dataset description what they're debarred from.